src/main/scala/io/iohk/ethereum/nodebuilder/StdNode.scala

```
@@ -62,6 +62,7 @@ abstract class BaseNode extends Node {
 
     startPeerManager()
 
+    startPortForwarding
```

Since this is side effecting, it should have `()` appended to it. I'd also recommend adding some logging somewhere so allocation can't just silently fail.

---

With the `AtomicReference` this is also fixable :+1: